### PR TITLE
Remove hardcoded url in bookworm (A solution to issue #8)

### PR DESCRIPTION
**Pull request by nctl144**
I implemented a demo solution to the issue #8 . It might not be the best solution but can you guys review it for me? I will try to make it better if you have any suggestion! I have free time after all :)

### CODE REVIEW

**Comment by kanav-raina** [comment]
hello nctl144,
will you participate in gsoc18 for this project .I am asking because i am also working on this project

**Comment by nctl144** [comment]
Thank you for pointing that out @lopuhin , I really appreciate that. I just fixed the solution to use the spider argument instead. Can you review it for me?

File: books/books/spiders/followall.py [hunk 1]
```diff
@@ -23,9 +23,10 @@ class FollowAllSpider(scrapy.Spider):
         'five': 5,
     }
 
-    def __init__(self, **kw):
+    def __init__(self, book_url=None, **kw):
         super(FollowAllSpider, self).__init__(**kw)
-        url = 'http://localhost/books.toscrape.com/index.html'
+
+        url = "%s" % book_url
```
Comment (lopuhin) [inline books/books/spiders/followall.py]: This could be ``url = book_url``, right?

**Comment by nctl144** [comment]
Yeah thank you for pointing that out! I don't even know why I wrote it like that because it looks kinda funny. Otherwise I changed it to the way you suggested @lopuhin

**Comment by lopuhin** [comment]
Thanks @nctl144 , nice job on fixing it!

**Pull request merged**